website/docs/r/metric_ruleset.html.markdown

```diff
@@ -62,5 +74,14 @@ The following arguments are supported in the resource block:
     * `dimensions` - (Required) List of dimensions to either be kept or dropped in the new aggregated MTSs
     * `drop_dimensions` - (Required) when true, the specified dimensions will be dropped from the aggregated MTSs
     * `output_name` - (Required) name of the new aggregated metric
+* `exception_rules` - (Optional) List of exception rules for the metric
```

Comment:
Do you think we want to add the description for `restoration` as well?

Reply:
I left it out because the UI doesn't support description.  However, considering our discussion on TF users generally not using UI, it's better to match the API spec 3.3.0.  I will add description to exception_rules.